hazelcast/src/test/java/com/hazelcast/partition/SafeClusterTest.java

```diff
@@ -31,68 +31,68 @@
 
 import static org.junit.Assert.assertTrue;
 
-// TODO tests are not sufficient. add more tests.
-
 /**
  * Includes tests for node and cluster safety.
+ *
+ * TODO: tests are not sufficient -> add more tests
  */
 @RunWith(HazelcastParallelClassRunner.class)
 @Category({QuickTest.class, ParallelTest.class})
 public class SafeClusterTest extends HazelcastTestSupport {
 
     @Test
-    public void isClusterSafe() throws Exception {
-        final HazelcastInstance node = createHazelcastInstance();
-        final boolean safe = node.getPartitionService().isClusterSafe();
+    public void isClusterSafe() {
+        HazelcastInstance node = createHazelcastInstance();
+        boolean safe = node.getPartitionService().isClusterSafe();
 
         assertTrue(safe);
     }
 
     @Test
-    public void isClusterSafe_multiNode() throws Exception {
-        final TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
-        final HazelcastInstance node1 = factory.newHazelcastInstance();
-        final HazelcastInstance node2 = factory.newHazelcastInstance();
-        final boolean safe1 = node1.getPartitionService().isClusterSafe();
-        final boolean safe2 = node2.getPartitionService().isClusterSafe();
+    public void isClusterSafe_multiNode() {
+        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
+        HazelcastInstance node1 = factory.newHazelcastInstance();
+        HazelcastInstance node2 = factory.newHazelcastInstance();
+        boolean safe1 = node1.getPartitionService().isClusterSafe();
+        boolean safe2 = node2.getPartitionService().isClusterSafe();
 
         assertTrue(safe1);
         assertTrue(safe2);
     }
 
     @Test
-    public void isLocalMemberSafe() throws Exception {
-        final HazelcastInstance node = createHazelcastInstance();
```

Comment:
Another final hater :)

Reply:
Noooooooooooooooo!!! :)

Reply:
I love it for fields, hate it for parameters and mostly dislike it for local variables (methods should be short enough to easily understand them without `final`).